website/content/docs/integrate/nia-integration.mdx

```diff
@@ -87,15 +87,15 @@ Once the module development has been completed another email should be sent to n
 
 At this stage, it is expected that the module is fully developed, all tests and documentation are in place, and that HashiCorp has reviewed the module to be compatible with Consul-Terraform-Sync.
 
-Once this is done, HashiCorp will get the new module listed as Consul-Terraform-Sync compatible on [consul.io](/docs/nia/installation/requirements#using-terraform-modules), and then the partner will be asked to publish the Terraform module to the [Terraform Registry](https://registry.terraform.io/browse/modules).
+Once this is done, HashiCorp will get the new module listed as Consul-Terraform-Sync compatible on [consul.io](/docs/nia/installation/requirements#partner-terraform-modules), and then the partner will be asked to publish the Terraform module to the [Terraform Registry](https://registry.terraform.io/browse/modules).
```

Comment:
Do we have any idea how "HashiCorp will get the new module listed [..]"? It would probably be better if we could document what they need to do to get the new module listed if we know what that is yet.

Reply:
I think the section before this captures what you're asking about. They'll need to engage with us to work through the approval process https://consul-git-cts-tf-modules-hashicorp.vercel.app/docs/integrate/nia-integration#4-review

Reply:
:+1: 
Missed it. Thanks for pointing it out.